src/data/index.ts

```diff
@@ -529,6 +530,13 @@ const json: RequirementsJson = {
       advisors: eceMinorAdvisors,
       abbrev: 'ECE',
     },
+    ECONILR: {
+      name: 'Economics for ILR Students',
```

Comment:
Another minor, information science in engineering, wrote the minoar like this
`Information Science [Engineering]`
should we keep the same semantics for consistency?

Reply:
Updated, good catch.